Use Go Modules.

d-kuro opened the pull request:
Use Go Modules can pin you to specific versions of dependencies to ensure stability in your systems.

skanehira commented:
Thanks your PR.

When using the Go module, is the tool installation and update as below?

```
git clone https://github.com/skanehira/docui.git
cd docui
GO 111 MODULE = on go install
```

d-kuro commented:
@skanehira 
The default for GO111MODULE is auto. Module support is enabled only when the current directory is outside GOPATH/src and itself contains a go.mod file or is below a directory containing a go.mod file. If you do `go get` as before, Module is not supported. you want to enable directory in GOPATH/src, specify GO111MODULE=on.

Go Modules support is in Go 1.11 and the new module paths are supported in Go 1.9.7+ and Go 1.10.3+ in a read-only. fully backward compatible because versions of Go older than 1.9.7 will ignore the mod file and import this package as they always have. So it does not force you to use Go Modules. You can install it the same way as before. Go Modules feel value in being able to specify the version of the library support.

* https://github.com/golang/go/wiki/Modules
* https://golang.org/cmd/go/

Thanks

friendbear commented on go.sum:
```diff
@@ -0,0 +1,63 @@
+github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 h1:w+iIsaOQNcT7OZ575w+acHgRric5iCyQh+xv+KJ4HB8=
+github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78/go.mod h1:LmzpDX56iTiv29bbRTIsUNlaFfuhWRQBWjQdVyAevI8=
```
Why Azure?

d-kuro replied:
go.sum is generated automatically.
result of [go mod graph](https://golang.org/cmd/go/#hdr-Print_module_requirement_graph) : 

```
$ GO111MODULE=on go mod graph
github.com/skanehira/docui github.com/fsouza/go-dockerclient@v1.3.3
github.com/skanehira/docui github.com/jroimartin/gocui@v0.4.0
github.com/skanehira/docui github.com/mattn/go-runewidth@v0.0.4
github.com/skanehira/docui github.com/nsf/termbox-go@v0.0.0-20190104133558-0938b5187e61
github.com/skanehira/docui github.com/skanehira/gocui-component@v0.0.0-20190103120946-5f5ba7f77c3b
github.com/skanehira/docui github.com/stretchr/testify@v1.3.0
github.com/skanehira/docui golang.org/x/sys@v0.0.0-20190115152922-a457fd036447
github.com/stretchr/testify@v1.3.0 github.com/davecgh/go-spew@v1.1.0
github.com/stretchr/testify@v1.3.0 github.com/pmezard/go-difflib@v1.0.0
github.com/stretchr/testify@v1.3.0 github.com/stretchr/objx@v0.1.0
github.com/fsouza/go-dockerclient@v1.3.3 github.com/Azure/go-ansiterm@v0.0.0-20170929234023-d6e3b3328b78
github.com/fsouza/go-dockerclient@v1.3.3 github.com/Microsoft/go-winio@v0.4.11
github.com/fsouza/go-dockerclient@v1.3.3 github.com/Nvveen/Gotty@v0.0.0-20120604004816-cd527374f1e5
github.com/fsouza/go-dockerclient@v1.3.3 github.com/containerd/continuity@v0.0.0-20181203112020-004b46473808
github.com/fsouza/go-dockerclient@v1.3.3 github.com/docker/docker@v0.7.3-0.20180827131323-0c5f8d2b9b23
github.com/fsouza/go-dockerclient@v1.3.3 github.com/docker/go-connections@v0.4.0
github.com/fsouza/go-dockerclient@v1.3.3 github.com/docker/go-units@v0.3.3
github.com/fsouza/go-dockerclient@v1.3.3 github.com/docker/libnetwork@v0.8.0-dev.2.0.20180608203834-19279f049241
github.com/fsouza/go-dockerclient@v1.3.3 github.com/gogo/protobuf@v1.2.0
github.com/fsouza/go-dockerclient@v1.3.3 github.com/google/go-cmp@v0.2.0
github.com/fsouza/go-dockerclient@v1.3.3 github.com/gorilla/context@v1.1.1
github.com/fsouza/go-dockerclient@v1.3.3 github.com/gorilla/mux@v1.6.2
github.com/fsouza/go-dockerclient@v1.3.3 github.com/opencontainers/go-digest@v1.0.0-rc1
github.com/fsouza/go-dockerclient@v1.3.3 github.com/opencontainers/image-spec@v1.0.1
github.com/fsouza/go-dockerclient@v1.3.3 github.com/opencontainers/runc@v0.1.1
github.com/fsouza/go-dockerclient@v1.3.3 github.com/pkg/errors@v0.8.1
github.com/fsouza/go-dockerclient@v1.3.3 github.com/sirupsen/logrus@v1.3.0
github.com/fsouza/go-dockerclient@v1.3.3 github.com/vishvananda/netlink@v1.0.0
github.com/fsouza/go-dockerclient@v1.3.3 github.com/vishvananda/netns@v0.0.0-20180720170159-13995c7128cc
github.com/fsouza/go-dockerclient@v1.3.3 golang.org/x/crypto@v0.0.0-20190103213133-ff983b9c42bc
github.com/fsouza/go-dockerclient@v1.3.3 golang.org/x/sys@v0.0.0-20190102155601-82a175fd1598
github.com/fsouza/go-dockerclient@v1.3.3 gotest.tools@v2.2.0+incompatible
github.com/sirupsen/logrus@v1.3.0 github.com/davecgh/go-spew@v1.1.1
github.com/sirupsen/logrus@v1.3.0 github.com/konsorten/go-windows-terminal-sequences@v1.0.1
github.com/sirupsen/logrus@v1.3.0 github.com/pmezard/go-difflib@v1.0.0
github.com/sirupsen/logrus@v1.3.0 github.com/stretchr/objx@v0.1.1
github.com/sirupsen/logrus@v1.3.0 github.com/stretchr/testify@v1.2.2
github.com/sirupsen/logrus@v1.3.0 golang.org/x/crypto@v0.0.0-20180904163835-0709b304e793
github.com/sirupsen/logrus@v1.3.0 golang.org/x/sys@v0.0.0-20180905080454-ebe1bf3edb33
```

friendbear replied:
@d-kuro 
Nice. it's best practice
thank's

d-kuro commented:
This Pull Requset  is Feel free to decline if this us unwanted.

d-kuro commented:
@skanehira 
Updated the README and wiki so please check it.
https://github.com/skanehira/docui/pull/27/commits/fcf2490f6f87f60bca72d446d50b8d982406de11

We added this procedure because the binaries created by `go get` or` go install` are in `$GOPATH/bin`

```
export PATH=$PATH:$GOPATH/bin
```

skanehira commented:
@d-kuro 
Thanks!
I have a comment so please check it.
